pygmt/base_plotting.py

```diff
@@ -223,6 +224,12 @@ def colorbar(self, **kwargs):
         scale : float
             Multiply all z-values in the CPT by the provided scale. By default
             the CPT is used as is.
+        shading : str or list or bool
+            Add illumination effects. Optionally, set the range of intensities
+            from -max_intens to +max_intens. If not specified, 1 is used.
+            Alternatively, set ``shading=[low, high]`` to specify an
+            asymmetric intensity range from *low* to *high*.
+            The default is no illumination.
```

Comment:
@willschlitzer The tests look good, but we may still have to rephrase these sentences because it's unclear to users what *max_intens* is.

Reply:
I think I made a change that makes it more clear.

Reply:
Looks great! Thanks.